rudder-web/src/main/scala/com/normation/rudder/web/snippet/node/Groups.scala

```diff
@@ -90,12 +90,12 @@ class Groups extends StatefulSnippet with DefaultExtendableSnippet[Groups] with
 
   val mainDispatch = {
     RudderConfig.configService.rudder_workflow_enabled match {
```

Comment:
We don't need this pattern matching anymore, since we do not use the value

Reply:
It's needed because of StatefulSnipper + used in nodeManagement/groups.html

Reply:
Do you still need the pattern matching ? just put mainDispatch = Map ( "head" .... "groupHierarchy" )

Reply:
AFAIK, it is till needed